### PR TITLE
Golbat support

### PR DESCRIPTION
**Pull request by unseenmagik**

### CODE REVIEW

File: trainer.sql [hunk 1]
```diff
@@ -1,81 +1,86 @@
...
+    `caught_steel`          int(6) UNSIGNED         DEFAULT NULL,
+    `caught_fire`           int(6) UNSIGNED         DEFAULT NULL,
+    `caught_water`          int(6) UNSIGNED         DEFAULT NULL,
+    `caught_grass`          int(6) UNSIGNED         DEFAULT NULL,
+    `caught_electric`       int(6) UNSIGNED         DEFAULT NULL,
+    `caught_psychic`        int(6) UNSIGNED         DEFAULT NULL,
+    `caught_ice`            int(6) UNSIGNED         DEFAULT NULL,
+    `caught_dragon`         int(6) UNSIGNED         DEFAULT NULL,
+    `caught_dark`           int(6) UNSIGNED         DEFAULT NULL,
+    `caught_fairy`          int(6) UNSIGNED         DEFAULT NULL,
```
Comment (na-ji) [inline trainer.sql:79]:
```suggestion
    `caught_fairy`          int(6) UNSIGNED         DEFAULT NULL
```

**Comment by unseenmagik** [comment]
Hi Nahi. 
I dont think there is anything else to add here now for original functionality to work with Golbat.
Ready for final review

**Comment by na-ji** [comment]
I will close the PR and re-open it inside the repo. This way, the CI will be fully working.

**Comment by na-ji** [comment]
Superseded by #436

**Pull request closed without merging**